### PR TITLE
Documentation

### LINKED ISSUE #3: Add JavaDoc
Add documentation to relevant classes (e.g. to `DemoPresenter`).

### PR DESCRIPTION
**Pull request by Steppschuh**
Resolves #3. Contains changes from #2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,4 +39,14 @@ nexenio_artifactory_password=PLACEHOLDER
 
 ## Getting Started
 
-Work in progress.
\ No newline at end of file
+You should start by checking out the Android library module called `sblecdemo`. It contains just 4 simple Moldel, View and Presenter classes:
```
Comment (Uwinator) [inline README.md]: please fix typo 'Moldel'

**Review by Uwinator — CHANGES_REQUESTED** [review]
There is a lot more than documentation in this PR. Please try to keep it separated, meaning for now: Handle the depending Wear stuff first, I'll review this PR afterwards

**Review by marvinmirtschin — APPROVED** [review]

**Comment by Steppschuh** [comment]
> Please try to keep it separated

I needed to add documentation to stuff that has was part of the other PR. What would you propose as the way to go in this case?

**Comment by Uwinator** [comment]
To make the review 'easy', either request the review when the first PR is merged, or use the dependent branch as target branch. With the second option, I would "blindly" merge it after both PRs are approved.

**Pull request merged**